packages/api/src/wallets/helpers/whatCryptoCurrencyType.ts

```diff
@@ -0,0 +1,7 @@
+import { CurrencyTypeEnum } from "@cryptify/common/src/enums/currency_type_enum";
+
+export function whatCryptoCurrencyType(walletAddress: string): CurrencyTypeEnum {
```

Comment:
This should probably also be in the common package

```suggestion
export function getCurrencyType(walletAddress: string): CurrencyTypeEnum {
```